games-emulation/dolphin/files/dolphin-libfmt-9.0.0.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/Source/Core/Common/MsgHandler.h b/Source/Core/Common/MsgHandler.h
```

Comment:
Is this pach upstream? Can you include a link to submission please?

Also, include a link to any relevant Gentoo bugs, if there are any.

Reply:
In my knowledge there is no gentoo bug yet, as the problem by now was resolved by pinning libfmt.
As the bug was resolved by a package pin i felt it might be better to really fix it rather then package pinning it.

This is my first contribution though so i might not be common with common rules

Reply:
You're absolutely right -- it's just that if there had been a bug, I'd want to tag it :)

It's fine that there isn't one here. What about patch link?

Reply:
https://github.com/Rudolchr/gentoo/blob/dolphin_libfmt-9_patch/games-emulation/dolphin/files/dolphin-libfmt-9.0.0.patch

if that is what i understand ur request :)

Reply:
Sorry, what I mean is:
- Who wrote it?
- If you wrote it, have you sent it upstream?
- If you didn't, where did you get it from?